searx/search.py

```diff
@@ -103,7 +107,8 @@ def search_one_offline_request(engine, query, request_params):
 
 def search_one_request_safe(engine_name, query, request_params, result_container, start_time, timeout_limit):
     if engines[engine_name].offline:
-        return search_one_offline_request_safe(engine_name, query, request_params, result_container, start_time, timeout_limit)  # noqa
+        return search_one_offline_request_safe(engine_name, query, request_params, result_container, start_time,
```

Comment:
is it possible to remove the new line here ?

Reply:
Done!